api/controller/triggers_test.go

```diff
@@ -47,6 +47,31 @@ func TestCreateTrigger(t *testing.T) {
 		So(resp.ID, ShouldResemble, triggerID)
 	})
 
+	Convey("Success with custom valid trigger", t, func() {
+		triggerID := "Valid_Custom_Trigger_Name-42"
+		triggerModel := dto.TriggerModel{ID: triggerID}
+		dataBase.EXPECT().GetTrigger(triggerModel.ID).Return(moira.Trigger{}, database.ErrNil)
+		dataBase.EXPECT().AcquireTriggerCheckLock(gomock.Any(), 10)
+		dataBase.EXPECT().DeleteTriggerCheckLock(gomock.Any())
+		dataBase.EXPECT().GetTriggerLastCheck(gomock.Any()).Return(moira.CheckData{}, database.ErrNil)
+		dataBase.EXPECT().SetTriggerLastCheck(gomock.Any(), gomock.Any(), gomock.Any()).Return(nil)
+		dataBase.EXPECT().SaveTrigger(gomock.Any(), triggerModel.ToMoiraTrigger()).Return(nil)
+		resp, err := CreateTrigger(dataBase, &triggerModel, make(map[string]bool))
+		So(err, ShouldBeNil)
+		So(resp.Message, ShouldResemble, "trigger created")
```

Comment:
JFYI `ShouldEqual` is enough, but it's ok, nevermind

Reply:
I used already existing tests as a template. It used `ShouldResemble` in the same case

Reply:
Ok, I did so also